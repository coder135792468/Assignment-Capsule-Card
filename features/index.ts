export * from "./capsule-container/capsule";
export * from "./header/header";
export * from "./pagination/pagination-button";
export * from "./utils";
export * from "./landing-page/landing";
