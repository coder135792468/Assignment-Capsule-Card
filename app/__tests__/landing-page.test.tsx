import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import { Landing } from "@/features/index";

describe("Landing Page", () => {
  it("renders a heading", () => {
    const heading = render(<Landing />);
    console.log("Found osmething", heading);
    expect(heading).toMatchSnapshot();
  });
});
