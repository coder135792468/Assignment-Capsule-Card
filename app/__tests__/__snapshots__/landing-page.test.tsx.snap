// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page renders a heading 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mb-10"
      >
        <div
          class="grid grid-cols-1 md:grid-cols-2 sm:grid-cols-1 xs:grid-cols-1 p-10 "
        >
          <img
            alt="Laning Image"
            class="md:w-[300px] md:h-[300px] sm:w-[250px] sm:h-[250px] xs:w-[200px] xs:h-[200px] mx-auto"
            data-nimg="1"
            decoding="async"
            height="40"
            loading="lazy"
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style="color: transparent;"
            width="40"
          />
          <div
            class="mt-10"
          >
            <h1
              class="font-medium text-[2rem]"
            >
              Capsule Card
            </h1>
            <ul
              class="list-disc ml-5 text-sm"
            >
              <li>
                Elevate your knowledges about capsules.
              </li>
              <li>
                Filter by status, original_launch, and type
              </li>
              <li>
                Accquire the information you need
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mb-10"
    >
      <div
        class="grid grid-cols-1 md:grid-cols-2 sm:grid-cols-1 xs:grid-cols-1 p-10 "
      >
        <img
          alt="Laning Image"
          class="md:w-[300px] md:h-[300px] sm:w-[250px] sm:h-[250px] xs:w-[200px] xs:h-[200px] mx-auto"
          data-nimg="1"
          decoding="async"
          height="40"
          loading="lazy"
          src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
          style="color: transparent;"
          width="40"
        />
        <div
          class="mt-10"
        >
          <h1
            class="font-medium text-[2rem]"
          >
            Capsule Card
          </h1>
          <ul
            class="list-disc ml-5 text-sm"
          >
            <li>
              Elevate your knowledges about capsules.
            </li>
            <li>
              Filter by status, original_launch, and type
            </li>
            <li>
              Accquire the information you need
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
